### ISSUE TITLE
[Crash] TypeError: Cannot destructure property 'channel' of 'props' as it is undefined.

### ISSUE THREAD

**Issue by C4J3** [arg0NNY/DiscordPlugins#114]
### Plugin:

📜 BetterChannelList

### What version of BetterDiscord are you using?

1.10.2

### What version of the selected plugin(s) are you using (if applicable)?

1.1.8

### What client are you using?

🔵 Stable

### Describe the bug

<details>
<summary>debug.log excerpt.</summary>

```
...
[2024-11-05 23:30:46.102][CONSOLE:LOG] [BetterChannelList] Requested last messages for guild 911237402612342814 channels: ["964517840575164446","911237916125167626","911242744238846032","1096721950279217174","911239830552334336","936266456893653033","924311139960102952","911312836922327110","1198290246018809856","1282622759884820571","1219946030502514688","1219948176606367784","1222470624107171840","1238484722934349874","1033683086602408026","1002887325535191120","1002887383139758110","1002887430929657926","1176093942094237716","1282621586033213463","1230209596421963868","1198282792878997541","1198282838643060819","1232999923713052703","1220654214422859837","1198282816992059482","1282627181712904223","1232742136722493571","1282626106805194795","1282626178313883698","1282626145698713685","1284430442509893757","1282626134479081522","1195034108972257411","1195034082405527593","1195034150139342959","964524490157617163","964524282594066532","992096802499207340","970365657013166131","967455153303224361","1060112219045642300","1060112807946887248","1060112088959303690","1084747039688101908","1061983226178056222","986709947221946439","986709817458569307","986709380978343966","1041845392049983608","959187980227084328","1130937755023716524","960974719341981766","911239533742419980","911239806095343616","911239545733918810","937597765733736469","914765998752690177"]
[2024-11-05 23:30:46.924][CONSOLE:ERROR] [BetterDiscord] [Patcher] Could not fire instead callback of Z for BetterChannelList Cannot destructure property 'channel' of 'props' as it is undefined.
TypeError: Cannot destructure property 'channel' of 'props' as it is undefined.
    at Object.eval [as callback] (betterdiscord://plugins/BetterChannelList.plugin.js:939:21)
    at betterdiscord/renderer.js:5:22680
    at W (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:944055)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1006892
    at a7 (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1006989)
    at ow (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1035662)
    at ok (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1036464)
    at oR (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1033531)
    at r6 (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:982257)
    at oD (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1034035)
    at t.flushSync (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1065158)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:8102048
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:8104363
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:2530779
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:2531568
    at Object.eG (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:952345)
    at eH (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:952499)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:970808
    at re (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:970907)
    at rt (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:971321)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:976783
    at oO (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1033804)
    at eM (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:951483)
    at ro (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:972612)
    at tG (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:958280)
    at tU (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:958180)
    at HTMLDivElement.E (https://discord.com/assets/sentry.889689cfe81d81d94ffd.js:1:760418)
[2024-11-05 23:30:46.933][CONSOLE:ERROR] [BetterDiscord] [Patcher] Could not fire instead callback of Z for BetterChannelList Cannot destructure property 'channel' of 'props' as it is undefined.
TypeError: Cannot destructure property 'channel' of 'props' as it is undefined.
    at Object.eval [as callback] (betterdiscord://plugins/BetterChannelList.plugin.js:939:21)
    at betterdiscord/renderer.js:5:22680
    at W (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:944055)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1006892
    at a7 (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1006989)
    at ow (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1035662)
    at ok (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1036464)
    at oR (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1033531)
    at r6 (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:982257)
    at oD (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1034035)
    at t.flushSync (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1065158)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:8102048
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:8104363
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:2530779
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:2531568
    at Object.eG (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:952345)
    at eH (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:952499)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:970808
    at re (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:970907)
    at rt (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:971321)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:976783
    at oO (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1033804)
    at eM (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:951483)
    at ro (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:972612)
    at tG (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:958280)
    at tU (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:958180)
    at HTMLDivElement.E (https://discord.com/assets/sentry.889689cfe81d81d94ffd.js:1:760418)
[2024-11-05 23:30:46.946][CONSOLE:ERROR] [BetterDiscord] [Patcher] Could not fire instead callback of Z for BetterChannelList Cannot destructure property 'channel' of 'props' as it is undefined.
TypeError: Cannot destructure property 'channel' of 'props' as it is undefined.
    at Object.eval [as callback] (betterdiscord://plugins/BetterChannelList.plugin.js:939:21)
    at betterdiscord/renderer.js:5:22680
    at W (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:944055)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1006892
    at a7 (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1006989)
    at ow (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1035662)
    at ok (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1036464)
    at oA (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1033197)
    at oR (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1033586)
    at r6 (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:982257)
    at oD (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1034035)
    at t.flushSync (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1065158)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:8102048
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:8104363
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:2530779
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:2531568
    at Object.eG (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:952345)
    at eH (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:952499)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:970808
    at re (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:970907)
    at rt (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:971321)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:976783
    at oO (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1033804)
    at eM (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:951483)
    at ro (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:972612)
    at tG (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:958280)
    at tU (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:958180)
    at HTMLDivElement.E (https://discord.com/assets/sentry.889689cfe81d81d94ffd.js:1:760418)
[2024-11-05 23:30:46.953][CONSOLE:ERROR] [BetterDiscord] [Patcher] Could not fire instead callback of Z for BetterChannelList Cannot destructure property 'channel' of 'props' as it is undefined.
TypeError: Cannot destructure property 'channel' of 'props' as it is undefined.
    at Object.eval [as callback] (betterdiscord://plugins/BetterChannelList.plugin.js:939:21)
    at betterdiscord/renderer.js:5:22680
    at W (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:944055)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1006892
    at a7 (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1006989)
    at ow (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1035662)
    at ok (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1036464)
    at oA (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1033197)
    at oR (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1033586)
    at r6 (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:982257)
    at oD (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1034035)
    at t.flushSync (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1065158)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:8102048
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:8104363
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:2530779
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:2531568
    at Object.eG (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:952345)
    at eH (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:952499)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:970808
    at re (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:970907)
    at rt (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:971321)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:976783
    at oO (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1033804)
    at eM (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:951483)
    at ro (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:972612)
    at tG (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:958280)
    at tU (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:958180)
    at HTMLDivElement.E (https://discord.com/assets/sentry.889689cfe81d81d94ffd.js:1:760418)
[2024-11-05 23:30:46.978][CONSOLE:ERROR] Cannot read properties of undefined (reading 'Messages')
TypeError: Cannot read properties of undefined (reading 'Messages')
    at ForumActivePostsCount (betterdiscord://plugins/BetterChannelList.plugin.js:650:23)
    at ah (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:995893)
    at l (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1053577)
    at oU (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1036572)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1036438
    at ok (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1036446)
    at oA (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1033197)
    at oR (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1033586)
    at r6 (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:982257)
    at oD (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1034035)
    at t.flushSync (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1065158)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:8102048
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:8104363
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:2530779
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:2531568
    at Object.eG (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:952345)
    at eH (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:952499)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:970808
    at re (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:970907)
    at rt (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:971321)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:976783
    at oO (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1033804)
    at eM (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:951483)
    at ro (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:972612)
    at tG (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:958280)
    at tU (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:958180)
    at HTMLDivElement.E (https://discord.com/assets/sentry.889689cfe81d81d94ffd.js:1:760418)
[2024-11-05 23:30:46.979][CONSOLE:ERROR] buildMessageReplyContent is not a function
TypeError: buildMessageReplyContent is not a function
    at ChannelLastMessage (betterdiscord://plugins/BetterChannelList.plugin.js:414:84)
    at ah (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:995893)
    at l (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1053577)
    at oU (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1036572)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1036438
    at ok (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1036446)
    at oA (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1033197)
    at oR (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1033586)
    at r6 (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:982257)
    at oD (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1034035)
    at t.flushSync (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1065158)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:8102048
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:8104363
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:2530779
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:2531568
    at Object.eG (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:952345)
    at eH (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:952499)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:970808
    at re (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:970907)
    at rt (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:971321)
    at https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:976783
    at oO (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:1033804)
    at eM (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:951483)
    at ro (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:972612)
    at tG (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:958280)
    at tU (https://discord.com/assets/web.c32565cdd9a2504c9bbb.js:12:958180)
    at HTMLDivElement.E (https://discord.com/assets/sentry.889689cfe81d81d94ffd.js:1:760418)
...
```
</details>

### Additional comments

Think BDFDB updated and might have broken something?

### I have read and agree to the following:

- [X] I have searched existing/closed issues to confirm that this hasn't been solved already.
- [X] I've already tried to disable all other plugins (or themes) to confirm this issue is related to the plugin.
This isn't necessarily required

**Comment by CorneliusCornbread** [comment]
Getting issues with this again too, certain servers will cause me to immediately crash

**Comment by sirlupusdev** [comment]
So this only happens in specific servers?

**Comment by CoolPeachGaming** [comment]
This happens across all servers that I've tried

**Comment by arg0NNY** [comment]
Fixed in 4806b85ca29fe14c8fe91cc9e5bc5bd93688100d
